packages/webapp/.eslintrc.cjs

```diff
@@ -17,6 +17,13 @@ const config = {
         "@typescript-eslint/no-empty-function": "off",
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/ban-ts-comment": "off",
+        "no-restricted-imports": "off",
+        "@typescript-eslint/no-restricted-imports": [
+            "error",
+            {
+                patterns: ["!./*", "!../*"],
+            },
+        ],
```

Comment:
So I was confused why this would work because this is the pattern used in the link I gave to ONLY ALLOW relative links.

Turns out:

- ` make check`  does not actually run the eslint checks, it needs to be `pnpm next lint --max-warnings 0`  and not ` pnpm eslint` (but let's actually call `make lint` from `make check`, that way we avoid duplication)

- Even so, this should forbid every single one of our imports... turns out it does not work, probably `!` is not a thing as an operator.

- BUT, if you remove the `!`, then it does exactly what we want, and indeed catches a few relative imports that slipped through the cracks :)

Can you fix `make check`, this file and eliminate the remaining relative imports? And also add a comment in this file to explain we forbid relative imports and absolute imports starting with `src/` should be used instead?

Oh, also you'll need to exclude `generated.ts` from eslint (`ignorePatterns` option).

I see we're also excluding `src/hooks/useScrollBox.ts` from linting, seems to be about React `useEffect` dependencies, would be good to remove it from the ignore list and either fix or simply add comments to disable eslint on those `useEffect` statements.